exercises/isogram/isogram_test.go

```diff
@@ -0,0 +1,50 @@
+package isogram
+
+import "testing"
+
+var testCases = []struct {
+	word     string
+	expected bool
+}{
+	{"duplicates", true},
+	{"eleven", false},
+	{"subdermatoglyphic", true},
+	{"Alphabet", false},
+	{"thumbscrew-japingly", true},
+	{"Hjelmqvist-Gryb-Zock-Pfund-Wax", true},
+	{"Heizölrückstoßabdämpfung", true},
+	{"the quick brown fox", false},
+	{"Emily Jung Schwartzkopf", true},
+	{"éléphant", false},
+}
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Errorf("Found testVersion = %v, want %v.", testVersion, targetTestVersion)
+	}
+}
```

Comment:
Can we put this at the bottom of the test file? The tests execute in parallel, so it doesn't technically matter where it is, but I since it's more like book-keeping than something essential to the problem, I think it's a bit nicer to keep it out of the way for the person reading it.